cookbook/generate_ligand_network.ipynb

```diff
@@ -20,25 +20,10 @@
   },
```

Comment:
<p>Here, as below, I'd like to execute all the code so our tests catch anything out of date.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/274/discussion/#comment-2608716914'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>